templates/steering-display/src/main.cpp

```diff
@@ -9,17 +9,26 @@
 #include <Arduino.h>
 #include <SPI.h>
 
+#include "can.hpp"
```

Comment:
Not specific to CAN but got me thinking. It might be worthwhile organizationally speaking to add an `include` folder to this template and move the headers there.

Reply:
Agreed. TODO